client/src/views/Home.view.tsx

```diff
@@ -48,22 +49,26 @@ const Home = () => {
       <div className="hidden md:block mt-30  h-auto w-auto">
         <div className="h-96 mt-40 mb-20 z-10">
           <MainText />
-            <a>
-              <button className="bg-[#FFC947] rounded-md  -mt-2 mb-10 text-black w-60 flex items-center absolute mt-10 h-12 left-1/2 transform -translate-x-1/2"
-                onClick={() =>
-                  (window.open('https://cyber-edu.co/register?token=40a121ba7d8ac969b5d08e041632f81ad4a599d11f054d4d1812ac94f480c4e34b4a9548de58bdf3', '_blank'))
-                }
-              >
-                <span className="mr-2 scale-125 absolute left-10 font-bold">
-                  Inscrie-te aici
+          <a>
+            <button
+              className="bg-[#FFC947] rounded-md  -mt-2 mb-10 text-black w-60 flex items-center absolute mt-10 h-12 left-1/2 transform -translate-x-1/2"
```

Comment:
here you give two values ​​for the same attribute, rewriting the first value
`-mt-2` and `mt-10` applies same css properties.